pkg/pod/status.go

```diff
@@ -281,6 +282,15 @@ func setTaskRunStatusBasedOnStepStatus(ctx context.Context, logger *zap.SugaredL
 				logger.Errorf("termination message could not be parsed as JSON: %v", err)
 				merr = multierror.Append(merr, err)
 			} else {
+				for _, r := range results {
+					if r.ResultType == result.ArtifactsResultType {
+						if err := json.Unmarshal([]byte(r.Value), &as); err != nil {
```

Comment:
NIT: in all other error cases, we have a `logger.Errorf` message, should we have one here too?

Reply:
sure I'll do it !